README.md

```diff
@@ -13,7 +13,7 @@ Built on the top of the `$http` service, Angular’s `$resource` is a service th
 ```js
 angular.module('app', [..., 'ngResource']);
 ```
-1. In the application directory run python -m SimpleHTTPServer 8000.
+1. In the application directory run `budo --open`.
```

Comment:
They used it yesterday. Taking 24's lead here.
https://github.com/mattdesl/budo